app/helpers/application_helper.rb

```diff
@@ -21,6 +21,9 @@ def meta_for(content)
     @keywords    = content.node.popular_tags.map &:name
     @description = content.title
     @dont_index  = true if content.node.score < 0
+    published_at = content.node.try(:created_at) || DateTime.now()
+    # For content recently published, set the index score threshold a bit higher
+    @dont_index ||= true if published_at > DateTime.now() - 24.hour && content.node.score < 5
```

Comment:
Use News.accept_threshold instead of 5?  (threshold a bit lower, newly published News will still be quickly indexed, no hard-coded value (at least not here))

Reply:
Indeed, it will be better to use a value which is not hard coded.

Although, the News.accept_threshold depends on the number of Account.amr accounts (admin + moderator) and I think we'll forgot on long term the link between the number of admin/moderators and the dont_index threshold.

What do you think about adding an application configuration like "NODE_DONT_INDEX_THRESHOLD = 5" in the `config/application.rb` ? We we'll be able to customize them in the `config/environments/production.rb` file if needed.